### ISSUE TITLE
1.0.3 has an error. options is now allowed limits.

### ISSUE THREAD

**Issue by breaklee** [expressjs/multer#202]
i setted 

```javascript
limits : {
		fileSize : 1024 * 1024 * 16  /* 16MB (bytes)*/
	}
```

if i upload big files than limits.

1.0.1 version
    error set with specific information

1.0.3 version
    error set null, and req.file also null

**Comment by LinusU** [comment]
@breaklee Could you try #205 and see if it fixes it for you?

**Comment by tsm91** [comment]
#205 doen't solve the file size limit bug, i tested your fix at #207

**Comment by kadishmal** [comment]
Today I have upgrade from 0.x to 1.0.3 and encountered the same error. Need a fix urgently.

**Comment by LinusU** [comment]
This should be fixed by #205 and released on npm as version `1.0.5` :rocket: 

Please reopen if issue persists.
